testfiles/RadLoHydrHeatCoolAuto.idf

```diff
@@ -1971,47 +1971,49 @@
     ,                        !- Number of Circuits
     ;                        !- Circuit Length {m}
 
-  Output:Variable,*,Zone Mean Air Temperature,timestep;
+  Output:Variable,*,Zone Mean Air Temperature, detailed;
```

Comment:
Thanks, @mitchute, for reviewing this PR!
Yeah, the input file changes were made during testing and debugging so that I could see what was going on all the time during the run.  I left them like this because I figured that it couldn't hurt, but the IDFs could also be flipped back to "timestep" if you would prefer that.  If so, let me know and I'll flip them back.  I'm okay with this either way and don't have any strong preference.

Reply:
I don't have an opinion either way. @Myoldmopar @mjwitte what's your preference?